build/export-exchanges.js

```diff
@@ -381,7 +381,8 @@ function exportKeywordsToPackageJson (exchanges) {
         keywords.add (ex.name)
     }
 
-    packageJSON.keywords = [...keywords]
+    const keywordsUnique = Object.values(Object.fromEntries(Array.from(keywords).map(s => [s.toLowerCase(), s])));
```

Comment:
Some of those are duplicated on purpose. I don't see any duplicates that would be rules out with the letter case in `package.json`. Can you please elaborate a bit more on this edit?

Reply:
i.e. these ones are repeated (with just different case):
```
    "Allcoin",
    "Bibox",
    "bitFlyer",
    "Bithumb",
    "Bitlish",
    "Braziliex",
    "BtcTrade.im",
    "bx.in.th",
    "cex.io",
    "Coingi",
    "Gate.io",
    "KuCoin",
    "MixCoins",
    "Novaexchange",
    "okcoin.cn",
    "okcoin.com",
    "SouthXchange",
    "DX.Exchange",
    "Eterbase",
    "Currency.com",
    "Waves.Exchange",
    "GOPAX",
    "Blockchain.com",
    "Crypto.com",
    "KuCoin Futures",
    "FMFW.io",
```